exercises/zipper/canonical-data.json

```diff
@@ -0,0 +1,715 @@
+{
+  "exercise": "zipper",
+  "version": "1.0.0",
+  "comments": [
+    " The test cases for this exercise include an initial tree and a     ",
+    " series of operations to perform on the initial tree.               ",
+    "                                                                    ",
+    " Trees are encoded as nested objects. Each node in the tree has     ",
+    " three members: 'value', 'left', and 'right'. Each value is a       ",
+    " number (for simplicity). Left and right are trees. An empty node   ",
+    " is encoded as null.                                                ",
+    "                                                                    ",
+    " Each operation in the operations list is an object. The function   ",
+    " name is listed under 'operation'. If the function requires         ",
+    " arguments, the argument is listed under 'item'. Some functions     ",
+    " require values (i.e.  numbers), while others require trees.        ",
+    " Comments are always optional and can be used almost anywhere.      "
+  ],
+  "cases": [
+    {
+      "description": "Creating a zipper for a binary tree",
+      "cases": [
+        {
+          "description": "data is retained",
+          "property": "property",
```

Comment:
I understand that there is no real good name for this property, but I hope there is at least a way to sneak a little bit of usefulness in here rather than leaving it as "property".

What of the suggestion of https://github.com/exercism/problem-specifications/blob/master/exercises/circular-buffer/canonical-data.json ("run") or https://github.com/exercism/problem-specifications/blob/master/exercises/react/canonical-data.json ("react")? Are they suitable for use here?

Reply:
I meant to decide on a good name but forgot :sweat_smile:
Do you have any suggestions?

Would we possibly use the property to categorize what kind of test we're running, i.e. `function(input) == output` vs. `function(input) == function(input)`?